google/cloud/spanner/samples/samples.cc

```diff
@@ -139,6 +139,48 @@ void DropDatabase(std::vector<std::string> const& argv) {
   (argv[0], argv[1], argv[2]);
 }
 
+void InsertData(std::vector<std::string> const& argv) {
+  if (argv.size() != 3) {
+    throw std::runtime_error(
+        "insert-data <project-id> <instance-id> <database-id>");
+  }
+
+  // TODO(#377) - cache the client across sample functions.
+  namespace gcs = google::cloud::spanner;
+  gcs::Client client(
+      gcs::MakeConnection(gcs::MakeDatabaseName(argv[0], argv[1], argv[2])));
+
+  //! [START spanner_insert_data]
+  namespace gcs = google::cloud::spanner;
+  [](gcs::Client client) {
+    auto insert_singers = gcs::InsertMutationBuilder(
+                              "Singers", {"SingerId", "FirstName", "LastName"})
+                              .EmplaceRow(1, "Marc", "Richards")
+                              .EmplaceRow(2, "Catalina", "Smith")
+                              .EmplaceRow(3, "Alice", "Trentor")
+                              .EmplaceRow(4, "Lea", "Martin")
+                              .EmplaceRow(5, "David", "Lomond");
```

Comment:
What do you think about ending this fluent call sequence with a call to `.Build();`? That would remove the need for `std::move(x).Build()` below.

Reply:
Done.